maestro-ios-driver/src/main/kotlin/util/XCRunnerCLIUtils.kt

```diff
@@ -44,7 +44,7 @@ object XCRunnerCLIUtils {
         val mapper = jacksonObjectMapper()
         val appsMap = mapper.readValue(json, Map::class.java) as Map<String, Any>
 
-        return appsMap.keys
+        return appsMap.keys + runningApps(deviceId).keys
```

Comment:
Included the runningApps as well since appsMap was not giving all the list of installed apps.